src/data/git-quiz.ts

```diff
@@ -467,7 +467,7 @@ const gitQuiz = [
       "Git is a popular version control system and GitHub is a hosting service for Git repositories.",
     Distractor1:
       "GitHub is a type of version control system, while Git is a hosting service for Git repositories.",
-    Distractor2: "Git and GitHub are the same thing",
+    Distractor2: "Git and GitHub are the same things",
```

Comment:
this change is extraneous for this PR, please remove it

Reply:
Removed.